const.go

```diff
@@ -122,16 +158,15 @@ var ConfToEnv = map[string]string{
 	// [SecretStore.Authentication] -- not supported
 	// [SecretStoreExclusive] -- application service only; not supported
 
+	// TODD: replace [Binding] with [Trigger] hierarchy
 	// [Binding]
-	"binding.type":            "app/BINDING_TYPE",
-	"binding.subscribe-topic": "app/BINDING_SUBSCRIBE_TOPIC",
-	"binding.publish-topic":   "app/BINDING_PUBLISH_TOPIC",
-
+	//"binding.type":            "app/BINDING_TYPE",
+	//"binding.subscribe-topic": "app/BINDING_SUBSCRIBE_TOPIC",
+	//"binding.publish-topic":   "app/BINDING_PUBLISH_TOPIC",
```

Comment:
Why are these commented out?

Reply:
I commented them out for now as `[Binding]` is not used anymore, but has been replaced by `[Trigger]` - see above